msgp_bench_test.go

```diff
@@ -0,0 +1,127 @@
+package capnp_test
+
+import (
+	"bytes"
+	"io"
+	"log"
+	"math/rand"
+	"testing"
+	"time"
+
+	capnp "zombiezen.com/go/capnproto2"
+	air "zombiezen.com/go/capnproto2/internal/aircraftlib"
+
+	"github.com/tinylib/msgp/msgp"
+)
+
+//go:generate msgp -tests=false -o msgp_bench_gen_test.go
+//msgp:Tuple Event
+
+type Event struct {
+	Name     string
+	BirthDay time.Time
+	Phone    string
+	Siblings int
+	Spouse   bool
+	Money    float64
+}
+
+func BenchmarkUnmarshalMsgp(b *testing.B) {
```

Comment:
Mind putting the MsgPack-related files and benchmarks behind a build tag?  I don't want the tests to depend on MsgPack unless specifically requested.